### ISSUE TITLE
[ENHANCE] Fast Return Protocol/emergency recall point

### ISSUE THREAD

**Issue by AlienXtream** [DarkholmeTenk/TardisModIssues#286]
a lever on the side of the console. only operable while the tardis has insufficient energy to travel inter dimensional. puts the tardis in a powered down state(cant fly, time locked to night, rondels stop functioning) for a period after use. intended as a last resort when stuck in a dimension you didn't intend to go to

**Comment by handicraftsman** [comment]
+

**Comment by DarkholmeTenk** [comment]
What does it do? Just reset you to the overworld or something?

**Comment by AlienXtream** [comment]
yeh. in an emmergency. and because it bypasses the energy requirement she shuts down to do a full recharge (like in FWV)

**Comment by SomeBadGamer** [comment]
Seems cool but a bit  cheaty because you could just use that instead of waiting for the recharge

**Comment by DarkholmeTenk** [comment]
It'll set your energy to 0 and disable flight until your energy is full again probably

**Comment by AlienXtream** [comment]
cool, that works :)

**Comment by AlienXtream** [comment]
and maybe force unstable flight on fastest speed?

**Comment by FoxPotato** [comment]
isn't flight already unstable on fastest flight. with the whole minigame getting harder? so just keep that in and you're set.

**Comment by AlienXtream** [comment]
no, the unstablized flight mode (autopilot off)

**Comment by TheAmazingSpy** [comment]
Basically, what I see is that everything shutdowns (roundels, anti-gravs, TARDIS flight manual, schematics, upgrading, everything shuts down) and goes dark, and then the TARDIS will teleport to where you first placed it or add a new option in the save/load place on the console that sets your 'emergency recall point' but if that doesn't work, it will teleport to your last destination you went to, like the gmod addon

**Comment by AlienXtream** [comment]
likely it will just pick a random location in the spawn chunks. and disabling the grav lifts is not a good idea imo cos then you can get stuck

**Comment by TheAmazingSpy** [comment]
But in the show, when the TARDIS has no power, the anti-grave are off, but maybe we can have hatches like FWV

**Comment by FoxPotato** [comment]
I agree with alien, turning off the grav lifts is a bad idea. it would only fuck people over.

**Comment by TheAmazingSpy** [comment]
How about add ladders, as the anti-gravs have emergency ladders, I'm only going with TARDIS Canon (not Doctor Who) because when the TARDIS needs more energy, it usually opens the Eye Of Harmony and sometimes turns off the anti-gravs because anti-gravs would use energy, this isn't magic

**Comment by FoxPotato** [comment]
i would be more suprised if ladders suddenly appeared than if the grav lifts still worked :p

**Comment by TheAmazingSpy** [comment]
No they're usually there, and somewhere in The Doctor's Wife, I think the climbed an anti-grave shaft after the TARDIS power was off

**Comment by AlienXtream** [comment]
tardis power was on. antigravs were turned off by "house". and FYI this IS kinda magic. "any sufficiently advanced technology is indistinguishable from magic". a gameboy would be magic to a caveman ;) (though to be fair a caveman likely wouldn't even be capable of recognizing that but you get the point

**Comment by FoxPotato** [comment]
I think a better example would be like a plane or something, but we get what you mean :P 
And who says the antigravs don't have like an internal capacitor. I mean, you can move them around with ease, without applying any wiring.

**Comment by TheAmazingSpy** [comment]
Fine, but the idea of turning the roundels off so they don't emit light and a small auxiliary lighting system comes on is good

**Comment by TheAmazingSpy** [comment]
Maybe if he adds a Eye Of Harmony Room it can open up, but it would be very unstable

**Comment by FoxPotato** [comment]
#305 dark explains it's not really possible to change the light lever roundels give off :p 
And the eye of harmony has been suggested many times, dark is thinking of maybe adding it. But it's porbably somewhere on the bottom of the to-do list.

**Comment by SomeBadGamer** [comment]
With the gravity  lifts maby he can make it work like the lab takes energy from the tardis/battery to work no/low energy  turned them  off  and to prevent  getting stuck tp any players  in the tardis  to the main console not entirely  sure  if that part is possible  but still worth a shot

**Comment by FoxPotato** [comment]
But if you get sent back to the console room anyway, doesn't that kinda defeat the point?

**Comment by SomeBadGamer** [comment]
True but it's better  than getting  stuck someware  high with no way down but death

**Comment by FoxPotato** [comment]
Yeah but, if you get teleported back anyway. Why bother turning off the gravity lifts. The whole thing was that you had to like put some effort in returning to the console. But if you get teleported back, it doesn't make that much sense to disable them.
